### DISCUSSION TITLE
Question about the input parameter yaw_angles_baseline

### DISCUSSION THREAD

**Discussion by Edward-Ernest** [NatLabRockies/floris#740] [Q&A]
Hello everyone,
      I am new to use Floris to do some research about wake and wind farm. Recently, I did some try with the optimization tools, but I can't understand the meanings of the input parameter `yaw_angles_baseline`. I know that `yaw_angles_baseline` is used to calculate the initial and baseline power production. But will it have influence to the output yaw angles optimized? For example, if I set `yaw_angles_baseline `as 50 deg and  wind_directions as 260 deg for one-turbine wind farm model, which one of these two angles will be relative to the optimized yaw angles?

**Comment by paulf81** [comment]
So `yaw_angles_baseline` is just used to calculate the baseline power.  The yaw angles are specified relative to the wind direction. Having a wind direction as 260 and yaw angle of 0 would mean the nacelle heading is 260 deg.  So that is why the default assumption is that the yaw angles are 0 for all turbines and all directions, which approximates normal operation.
